src/PKSim.Core/Snapshots/Mappers/SimulationMapper.cs

```diff
@@ -181,7 +181,9 @@ private IEnumerable<IParameter> parametersChangedFromBuildingBlock(IGrouping<PKS
 
       private static bool parameterDiffersFromTemplate(IParameter templateBuildinglockParameter, IParameter parameter)
       {
-         return templateBuildinglockParameter == null || !ValueComparer.AreValuesEqual(parameter, templateBuildinglockParameter);
+         return templateBuildinglockParameter == null || 
+                !templateBuildinglockParameter.ValueIsDefined() ||
```

Comment:
This is a bug that was introduce when trying to not export too many paraemters. Sometimes, parameter in BB cannot be evaluated. Need to check that before comparing